site/content/extend/plugins/server/best-practices.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Best Practices
+date: 2018-07-10T00:00:00-05:00
+subsection: Server Plugins
+weight: 0
+---
+
+## How should server plugins handle static files?
```

Comment:
The title is a bit ambiguous. There two cases if static files:
1. The plugins wants to load a static file and use it from the code
2. The file should be publicly available via http.

The text you proposed deals with 2. But the title doesn't clearly state this.

Reply:
What do you think about: `How should plugins serve publicly available static files`?

Reply:
Sounds good to me 👍